Node/samples/app.js

```diff
@@ -145,13 +149,29 @@ bot.dialog('StartNewReplyChain', function (session) {
 });
 bot.dialog('MentionUser', function (session) {
     // user name/user id
-    var toMention = {
-        name: 'Bill Zeng',
-        id: userId
+    var user = {
+        id: userId,
+        name: 'Bill Zeng'
     };
-    var msg = new teams.TeamsMessage(session).text(teams.TeamsMessage.getTenantId(session.message));
-    var mentionedMsg = msg.addMentionToText(toMention);
-    session.send(mentionedMsg);
+    var mention = new teams.UserMention(user);
+    var msg = new teams.TeamsMessage(session).addEntity(mention).text(mention.text + ' ' + teams.TeamsMessage.getTenantId(session.message));
+    session.send(msg);
+    session.endDialog();
+});
+bot.dialog('MentionChannel', function (session) {
```

Comment:
Do you have an example of MentionTeam? I should clarify that this feature also allows bots to mention team (with "type" set to "team")

Reply:
Let's focus on app.ts file

Reply:
Added